travis-emcc-tests.sh

```diff
@@ -1,15 +1,10 @@
 #!/usr/bin/env bash
 
 set -e
-echo "travis-test build:wasm"
+echo "travis-test build"
 emconfigure cmake -DCMAKE_BUILD_TYPE=Release
 emmake make -j4 binaryen_wasm
-echo "travis-test test:wasm"
-python3 -m scripts.test.binaryenjs
-echo "travis-test done:wasm"
-
-echo "travis-test build:js"
 emmake make -j4 binaryen_js
-echo "travis-test test:js"
+echo "travis-test test"
 python3 -m scripts.test.binaryenjs
```

Comment:
the name becomes confusing now, i worry - we build `binaryen_js` and `binaryen_wasm`, then run `test.binaryenjs` to test them both? Perhaps the test name could be be different, like `test_binaryen_js_and_wasm`?

Reply:
Last commit now makes it so that one can specify an additional argument `js` to test the JS variant

```
python3 -m scripts.test.binaryenjs js
```

respectively `wasm` to test the Wasm variant

```
python3 -m scripts.test.binaryenjs wasm
```

and updates the travis script to build and test both in order. Not specifying the additional argument still tests both. That ok?